src/main/java/org/softserve/edu/utils/MavenPropertiesFileReader.java

```diff
@@ -0,0 +1,33 @@
+package org.softserve.edu.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class MavenPropertiesFileReader {
+
+    private static final Logger LOG = LogManager.getLogger(FileUtils.class.getName());
+
+    public static void main(String[] args) {
```

Comment:
Use the static method to get maven property:
Example: MavenPropertiesFileReader.getMavenProperty("UsedEnvironment"); // result should be local or qa

Reply:
Made them static